scripts/setup_network_storage.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import stat
+import time
+
+import shutil
+from pathlib import Path
+from concurrent.futures import as_completed
+from addict import Dict as NSDict
+
+import util
+from util import (
+    lkp,
+    dirs,
+)
+
+
+def mounts_by_local(mounts):
+    """convert list of mounts to dict of mounts, local_mount as key"""
+    return {str(Path(m.local_mount).resolve()): m for m in mounts}
+
+
+def resolve_network_storage(nodeset=None):
+    """Combine appropriate network_storage fields to a single list"""
+
+    cfg = util.load_config_file(Path(__file__).with_name("config.yaml"))
```

Comment:
Why are you not just importing config as it was done before? To me this change make it where this is not a no-op.

Also this is the kind of thing that should go in a separate commit.

Reply:
Gotcha, agree and makes sense. Updated to keep it as it is.